packages/core/lib/src/dummy_data/dummy_media.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:core/src/models/media.dart';
+
+/// Holds dummy media used for testing and the Widgetbook app
+class DummyMedia {
```

Comment:
@jenshor This might not be very clean or optimal and I'm happy to discuss it. 
This dummy data will be used only in tests and in `app_widgetbook`. I usually add this inside the `test` directory but here I think accessing it from `app_widgetbook` helps populate widgets with data quickly.
What do you think?

Reply:
Talked about this. We can have a bit of redundancy so we don't export test data in the package